lib/espree.js

```diff
@@ -174,6 +174,9 @@ module.exports = () => Parser => class Espree extends Parser {
             this.next();
         } while (this.type !== tokTypes.eof);
 
+        // it's a call for the latest `eof` token
```

Comment:
I think the word "latest" is confusing here because it looks like `this.next()` will continue returning muliple `eof` tokens. I wonder if something like the following would be clearer:

```suggestion
        // Consume the final eof token
```